docs/root/configuration/listeners/listener_filters/original_src_filter.rst

```diff
@@ -3,7 +3,7 @@
 Original Source
 ===============
 
-* :ref:`Listener filter v2 API reference <envoy_api_msg_config.filter.listener.original_src.v2alpha1.OriginalSrc>`
+* :ref:`Listener filter v3 API reference <envoy_v3_api_msg_extensions.filters.listener.original_src.v3.OriginalSrc>`
```

Comment:
How did you determine that all filters/extensions are covered? Are there no "v2 API" references left via grep?

Reply:
Ran my script. manually checked each file for v2 references.

Reply:
I would be nice to drop the references from the v2 API generated docs so then we would get errors, but then cross links would break. I wonder if that is OK? Maybe something else to track / think about for tooling follow ups.

Reply:
I think we could have a build mode that removed all v2 cross links and refs and get this property. Best to punt to a future issue here.